Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/server/CoronaTestResult.kt

```diff
@@ -90,3 +90,15 @@ enum class CoronaTestResult(val value: Int) {
         }
     }
 }
+
+val CoronaTestResult?.isFinalResult
+    get() = when (this) {
+        CoronaTestResult.PCR_POSITIVE,
+        CoronaTestResult.PCR_NEGATIVE,
+        CoronaTestResult.RAT_POSITIVE,
+        CoronaTestResult.RAT_NEGATIVE -> true
+        else -> false
+    }
+
+val CoronaTestResult?.isPending
```

Comment:
and rat pending?

Reply:
The state says RAT_OR_PCR_PENDING, and this is not new, it was just moved  from one place to another 
see https://github.com/corona-warn-app/cwa-app-android/pull/4986/files#diff-74d8a3af0a20d2b26b9c921ae7b3bbf64dd14995365a3d31f7b3b483d7f970d9L76-L87